packages/design-system-tokens/src/themes/healthcare-components.ts

```diff
@@ -241,6 +241,16 @@ export const components: AllTokenValues = {
     '__border-radius':                            '0',
   },
 
+  'usaBanner': {
```

Comment:
Have we used `camelCase` on these names before? Looks like the one below uses `kebab-case`

Reply:
Need to switch that over to camel case as well, for parsing these later it's easier to split on `-` so camel case keeps things better organized.

Reply:
For now neither is needed but I will clean it up in a big once-over in the main branch before this feature branch gets merged.